### PR TITLE
Some corrections and updates done.

### PR DESCRIPTION
**Pull request by AdityaGarg8**

### CODE REVIEW

File: docs/state.md [hunk 1]
```diff
@@ -9,13 +9,13 @@ While Linux is usable on all T2 models, some features are limited by lack of dri
 - USB
 - Keyboard
 - Camera
+- Wifi
```
Comment (networkException) [inline docs/state.md:12]: Some models do not have support, notably the 15,4

**Review by networkException — CHANGES_REQUESTED** [review]

File: docs/state.md (hunk 1 above)
Comment (Redecorating) [reply docs/state.md:12]: The ones without working wifi have the BCM4377 chip: MacBookPro15,4, MacBookPro16,3, MacBookAir9,1 and possibly MacBookAir8,2 (idk which chip that one has)

Comment (networkException) [reply docs/state.md:12]: I'd still rather have that information in the wifi guide

Comment (AdityaGarg8) [reply docs/state.md]: Done the corrections

**Comment by networkException** [comment]
Would you be so kind to rebase any future pull request on top of our master branch? The pull request descriptions are getting confusing

**Pull request merged**